src/components/hand-poses.js

```diff
@@ -11,13 +11,31 @@ const POSES = {
 
 const NETWORK_POSES = ["allOpen", "thumbDown", "indexDown", "mrpDown", "thumbsUp", "point", "allGrip", "pinch"];
 
-AFRAME.registerComponent("hand-pose", {
+AFRAME.registerComponent("hand-pose-state", {
   multiple: true,
   schema: {
     pose: { default: 0 }
   },
+  init() {
+    this.setSelfAsStore = this.setSelfAsStore.bind(this);
+  },
+  play() {
+    this.el.addEventListener("model-loaded", this.setSelfAsStore);
+  },
+  pause() {
+    this.el.removeEventListener("model-loaded", this.setSelfAsStore);
+  },
+  setSelfAsStore(e) {
+    let poseEl = e.target.querySelector(`[hand-pose__${this.id}]`);
+    poseEl.components[`hand-pose__${this.id}`].store = this;
```

Comment:
I'm confused by this, can't the hand pose just look up this element itself via its selector?